code/game/machinery/portable_turret.dm

```diff
@@ -902,7 +902,7 @@ Status: []<BR>"},
 
 /obj/machinery/porta_turret_cover/attack_hand(mob/user)
 	. = ..()
-	if(.)
+	if(. || !Parent_Turret)

```

Comment:
Well dreamstation has 4 turret covers in the AI core without turrets attached to them, so mapping errors might be the source of this